src/app/pages/network/configuration/configuration.component.ts

```diff
@@ -64,19 +64,19 @@ export class ConfigurationComponent {
       type : 'input',
       name : 'gc_nameserver1',
       placeholder : T('Nameserver 1'),
-      tooltip : T('The primary DNS server, typically a Windows domain.'),
+      tooltip : T('Enter the primary DNS server.'),
     },
     {
       type : 'input',
       name : 'gc_nameserver2',
       placeholder : T('Nameserver 2'),
-      tooltip : T('Secondary DNS server.'),
+      tooltip : T('Enter the secondary DNS server.'),
     },
     {
       type : 'input',
       name : 'gc_nameserver3',
       placeholder : T('Nameserver 3'),
-      tooltip : T('Tertiary DNS server'),
+      tooltip : T('Enter the tertiary DNS server'),
```

Comment:
As above.  Suggest using "third" rather than "tertiary", also.

Reply:
I like the word tertiary. :) Changed to Third.